### DISCUSSION TITLE
docfx v3 - docfx.yml - How to add source projects?

### DISCUSSION THREAD

**Discussion by VaclavElias** [dotnet/docfx#8227] [Q&A] [answered]
I can't find an example, how to add source project to docfx **v3** when I am using **docfx.yml** to generate api docs.

The same question was asked in the past https://github.com/dotnet/docfx/issues/3958. There is not a clear answer and referenced link to spec doesn't provide the answer (if I am right).

Please, could you advise how can I do that? I am using currently this branch https://github.com/dotnet/docfx/tree/v3 which I am working with and I am able to generate *.md pages.

[Example](https://dotnet.github.io/docfx/tutorial/docfx.exe_user_manual.html#3-docfxjson-format) v2
```
{
  "metadata": [
    {
      "src": [
        {
          "files": ["**/*.csproj"],
          "exclude": [ "**/bin/**", "**/obj/**" ],
          "src": "../src"
        }
      ],
      "dest": "obj/docfx/api/dotnet",
      "shouldSkipMarkup": true,
      "properties": {
          "TargetFramework": "netstandard1.3"
      }
    }]
}
```

How can I do the same in v3 using  **docfx.yml**?

Thanks 🙂

**Comment by paulushub** [comment, marked as answer]
I am not sure V3 supports source project. They are moving to MonoDoc so use of Rosyln may not be needed!

**Reply by VaclavElias** [reply]
I see, I shall research MonoDoc then 🙂

**Reply by VaclavElias** [reply]
Hmm, there is not too much written, I have found some old blog posts https://den.dev/blog/summer-of-docs-net/ but things have changed since that.

I managed to figure it out, it is using `mdoc` as you mentioned and then docfx v3 kind of takes care of the rest and the result is here.

I have to try to get a sense of it all and maybe to write an updated blog post..

Thanks again for you help @paulushub 

<img width="955" alt="image" src="https://user-images.githubusercontent.com/4528464/198882832-99cb7455-125e-44cb-810e-d078b1533b88.png">
